go/README.md

```diff
@@ -1,10 +1,9 @@
 ## fabric-admin-sdk for golang
 
-## folder structure
-- chaincode
-- channel
-- tools
-- e2e
-
-## test
-use ginkgo for testing
\ No newline at end of file
+### [Overview](https://github.com/hyperledger/fabric-rfcs/pull/55)
+
+### [Features](https://github.com/Hyperledger-TWGC/fabric-admin-sdk/issues/15)
+
+### [Scenario tests and example](./test/e2e_test.go)
+
+### [Documentation](https://pkg.go.dev/search?q=fabric-admin-sdk&m=package)
```

Comment:
Once the module path is correct, this link should look something like:
```
https://pkg.go.dev/github.com/hyperledger-twgc/fabric-admin-sdk
```
or
```
https://pkg.go.dev/github.com/hyperledger-twgc/fabric-admin-sdk/go
```

Reply:
change to `https://pkg.go.dev/github.com/hyperledger/fabric-admin-sdk` as ideally